cypress/support/commands/assessment.ts

```diff
@@ -1,9 +1,9 @@
 // commands related to the overall assessment
 
 export const saveAndContinue = () => {
-  return cy.get(`form button`).should('contain.text', 'Save and continue').click()
+  return cy.get(`form button:contains('Save and continue')`).should('have.length', 1).click()
 }
 
 export const markAsComplete = () => {
-  return cy.get(`form button`).should('contain.text', 'Mark as complete').click()
+  return cy.get(`form button:contains('Mark as complete')`).should('have.length', 1).click()
```

Comment:
Mind explaining the logic behind this? Was there an issue where multiple of the button would appear?